cli/src/factory_ops/unboxing/network/public_mgmt

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+
+set -euE
+export NETWORK_MANAGEMENT=1
+
+LOG_FILE="${LOG_FILE:-/var/log/seagate/provisioner/unboxing_nw_update.log}"
+mkdir -p $(dirname "${LOG_FILE}")
+#truncate -s 0 ${LOG_FILE}
+
+PRVSNR_ROOT="/opt/seagate/cortx/provisioner"
+SALT_OPTS="--no-color --out-file=${LOG_FILE} --out-file-append"
+
+pvt_ip_a=$(get_pillar_data cluster:srvnode-1:network:data_nw:pvt_ip_addr)
+pvt_ip_b=$(get_pillar_data cluster:srvnode-2:network:data_nw:pvt_ip_addr)
+
+remotes=(
+    ${pvt_ip_a}
+    ${pvt_ip_b}
+)
+
+
+function backup_ifcfg {
+    node=${1:-${pvt_ip_a}}
+    if_name=${2:-}
+
+    if [[ -n if_name ]]; then
+        if [[ ! $(ssh_over_pvt_data ${node} "test -e /etc/sysconfig/network-scripts/ifcfg-${if_name}.boxing" >/dev/null 2>&1) ]]; then
+            _linfo "Backing up ifcfg-${if_name} to ifcfg-${if_name}.boxing in dir /etc/sysconfig/network-scripts"
+            ssh_over_pvt_data ${node} "cp /etc/sysconfig/network-scripts/ifcfg-${if_name} /etc/sysconfig/network-scripts/ifcfg-${if_name}.boxing"
+        else
+            _lerror "ifcfg-${if_name}.boxing file from previous boxing already exists. Skipping backup..."
+        fi
+    fi
+}
+
+
+function test_mgmt_ips {
+    _linfo "Fetching IP of the nodes"
+    remote_mgmt_ip_srvnode_1=$(ssh_over_pvt_data ${pvt_ip_a} "ip -4 address show dev eno1 | grep inet | head -1 | awk '{print \$2}' | awk -F '/' '{print \$1}'")
+    remote_mgmt_ip_srvnode_2=$(ssh_over_pvt_data ${pvt_ip_b} "ip -4 address show dev eno1 | grep inet | head -1 | awk '{print \$2}' | awk -F '/' '{print \$1}'")
+    
+    # Test the IP that is set
+    _linfo "Ping management IP ${remote_mgmt_ip_srvnode_2} of Server-B from Server-A to test sanity."
+    ping -c1 ${remote_mgmt_ip_srvnode_2} && (
```

Comment:
@ypise  shoudn't we use `-I` here?

Reply:
Not really required for management network as it is default, but having it would not harm.